wwwapp/models.py

```diff
@@ -59,6 +59,47 @@ class WorkshopUserProfile(models.Model):
     def __str__(self):
         return '%s: %s, %s' % (self.year, self.user_profile, self.status)
 
+class PESELField(models.CharField):
+    """PESEL field, with checksum verification."""
+
+    def validate(self, pesel: str, user_info: 'UserInfo') -> None:
+        if not pesel:
+            raise ValidationError('PESEL type is {})'.format(type(pesel)))
```

Comment:
This will only happen when the pesel length is 0, what is the point of this error? The type here is always a string (check `CharField.to_python()`)